src/interface/gui.py

```diff
@@ -0,0 +1,75 @@
+import streamlit as st
+import os
+import fitz
+
+from streamlit_pdf_viewer import pdf_viewer
+from .initialize import initialize_variables
+
+
+class Gui:
+    def __init__(self, path_file: str, path_modified_file: str, message_container_height: int) -> None:

```

Comment:
Please get familiar with attrs library, how to write some classes without boilerplate